cmd/aiservices/deployment/deployment_delete.go

```diff
@@ -37,24 +37,51 @@ func (c *DeploymentDeleteCmd) CmdRun(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	// Resolve deployment ID using the SDK helper
+	// Resolve deployment IDs using the SDK helper
 	list, err := client.ListDeployments(ctx)
 	if err != nil {
 		return err
 	}
-	entry, err := list.FindListDeploymentsResponseEntry(c.Deployment)
-	if err != nil {
-		return err
+
+	deploymentsToDelete := []v3.UUID{}
+	for _, deploymentStr := range c.Deployments {
+		entry, err := list.FindListDeploymentsResponseEntry(deploymentStr)
+		if err != nil {
+			if !c.Force {
+				return err
+			}
+			fmt.Fprintf(os.Stderr, "warning: %s not found.\n", deploymentStr)
```

Comment:
Not always correct: https://github.com/exoscale/egoscale/blob/master/v3/operations.go#L28